migrations/20240321211311_event.up.sql

```diff
@@ -0,0 +1,18 @@
+-- Add up migration script here
+
+CREATE TABLE IF NOT EXISTS event (
+    id BLOB NOT NULL, -- network_id sort_value controller StreamID height event_cid
```

Comment:
do we want to normalize this to to something like the following to support a flat recon future? 

```sql
network_id integer not null,
sep_key text not null default 'model',
sep_value bytes not null,
controller text not null,
init_cid bytes not null
```

Reply:
decided against this. it makes queries a lot more complicated. whatever we do now will likely require a recon protocol version bump and we are still a bit from landing at the flat recon implementation.